src/act-compat.js

```diff
@@ -1,6 +1,12 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import {act as reactAct} from 'react-dom/test-utils'
+
+let reactAct
+try {
+  reactAct = require('react-dom/test-utils').act
```

Comment:
Will this throw if act is not defined but test-utils is?

Reply:
No it wont, but we'll handle it fine anyway thanks to the `||` below 👍